src/content/articles/kubernetes-for-scala-developers-part-one/index.mdx

```diff
@@ -140,77 +142,35 @@ spec:
      image: aidocking/pfpw:20.20.31
      ports:
        - containerPort: 9000
-
-# Deploy: `kubectl apply -f workload-api.yaml`
-# View: `kubectl get pods`
-# Drop: `kubectl delete -f workload-api.yaml`
 ```
-
-* API Version and Kind: The first two lines specify the Kubernetes Pods API version, which is crucial for Kubernetes
+Deploy: `kubectl apply -f workload-api.yaml`
```

Comment:
Just put the terminal command and output here instead of the image. Do this everywhere.

Reply:
Done